stable/weave-cloud/templates/manifest.yaml

```diff
@@ -0,0 +1,402 @@
+apiVersion: v1
+kind: List
+items:
+  - metadata:
+      labels:
+        name: weave-flux
+      name: weave-flux
+      annotations:
+        cloud.weave.works/launcher-info: |-
```

Comment:
What are these annotations and should they be hardcoded? Also would it help to put them in a file and include the file as they seem to be used on all resources.